exercises/practice/atbash/.meta/tests.toml

```diff
@@ -0,0 +1,13 @@
+# This is an auto-generated file.
+#
+# Regenerating this file via `configlet sync` will:
+# - Recreate every `description` key/value pair
+# - Recreate every `reimplements` key/value pair, where they exist in problem-specifications
+# - Remove any `include = true` key/value pair (an omitted `include` key implies inclusion)
+# - Preserve any other key/value pair
+#
+# As user-added comments (using the # character) will be removed when this file
+# is regenerated, comments can be added via a `comment` key.
+
+[2c794dd6-4779-4454-910e-f3aabe820a13]
+description = "Implement the Atbash cipher"
```

Comment:
This probably isn't updated correctly, I am not yet sure how to auto-generate this file. Not too familiar yet with the context of a language-track's repo within the overall exercism local setup so I hadn't managed to run `configlet sync` on this yet.

What's the recommended procedure to get the right UUID and auto-gen meta files such as this when adding a new exercise/solution?